source/educators/navigation/olx.rst

```diff
@@ -152,18 +152,18 @@ This section describes the various exercises, tools, and problem types that you
    /educators/references/course_development/exercise_tools/ORA_Staff_Grading.rst
```

Comment:
At the beginning of the file I'd change "This guide is intended for those interested in using OLX (open learning XML) to develop edX courses. Readers of this guide should be familiar with XML." to "develop *Open* edX courses"